index.js

```diff
@@ -86,4 +86,33 @@ document.querySelector('#book-form').addEventListener('submit', (e) => {
 document.querySelector('#book-collection').addEventListener('click', (e) => {
   ShowBooks.deleteBook(e.target);
   Archive.removeBook(e.target.parentElement.previousElementSibling.textContent);
-});
\ No newline at end of file
+});
+
+const navList = document.querySelector('.list');
+const navAddNew = document.querySelector('.add-new');
+const navContact = document.querySelector('.contact');
+
+const listSection = document.querySelector('#list');
+const AddNewSection = document.querySelector('#add-new');
+const contactSection = document.querySelector('#contact');
+
+navList.addEventListener('click', () => {
+  listSection.classList.remove('hide');
+  AddNewSection.classList.add('hide');
+  contactSection.classList.add('hide');
+});
+
+navAddNew.addEventListener('click', () => {
+  listSection.classList.add('hide');
+  AddNewSection.classList.remove('hide');
+  contactSection.classList.add('hide');
+});
+
+navContact.addEventListener('click', () => {
+  listSection.classList.add('hide');
+  AddNewSection.classList.add('hide');
+  contactSection.classList.remove('hide');
+});
+
+const today = document.querySelector('#date-time');
+today.innerHTML = new Date();
```

Comment:
Thank you for implementing the date feature :+1: 

- [ ] Although the current date feature is working, you're expected to use the Luxon library. 
- If you're experiencing any difficulty with the provided links on the project requirements page, check [this link](https://moment.github.io/luxon/#/)